packages/node/src/indexer/api.service.test.ts

```diff
@@ -96,3 +108,44 @@ describe.skip('ApiService', () => {
     expect(txInfos.length).toEqual(4);
   });
 });
+
+describe('ApiService Chihuahua', () => {
+  let app: INestApplication;
+  let apiService: ApiService;
+  const prepareApiService = async () => {
+    const module = await Test.createTestingModule({
+      providers: [
+        ConnectionPoolStateManager,
+        ConnectionPoolService,
+        {
+          provide: 'ISubqueryProject',
+          useFactory: () =>
+            testCosmosProject(ENDPOINT_CHIHUAHUA, CHAINID_CHIHUAHUA),
+        },
+        {
+          provide: NodeConfig,
+          useFactory: () => ({}),
+        },
+        EventEmitter2,
+        ApiService,
+        NodeConfig,
+      ],
+      imports: [EventEmitterModule.forRoot()],
+    }).compile();
+    app = module.createNestApplication();
+    await app.init();
+    apiService = app.get(ApiService);
+    await apiService.init();
+  };
+
+  beforeAll(async () => {
+    await prepareApiService();
+  });
+
+  it('chihuahua test, can query block result', async () => {
```

Comment:
It would be good to have a comment with details about the specific issue we faced